addon/webextension/catcher.js

```diff
@@ -58,6 +58,7 @@ window.catcher = (function () {
       console.error("------Error in promise:", e+"");
       console.error(e.stack);
       exports.unhandled(makeError(e));
+      throw e;
```

Comment:
Usually yes.  In the case I worked through here it causes the failure to go back to selector/shooter.js

It would probably be better if we were more explicit about expectations in each case but without doing that work I thought this would be a better default.